test cases/zig/2 sharedlib/meson.build

```diff
@@ -0,0 +1,4 @@
+project('zigtest-shared-lib', 'zig', version: '0.1.0')
+
+lib = shared_library('hello', 'lib/hello.zig')
```

Comment:
there's a couple things this test should also do, let's add a `version` to the shared lib, and then install both the shared lib and the executable. Then you can use a test.json file to ensure that the library is installed with the correct version information.